psi4/src/psi4/libmints/mintshelper.cc

```diff
@@ -1132,6 +1199,7 @@ SharedMatrix MintsHelper::mo_eri_helper(SharedMatrix Iso, SharedMatrix C1, Share
     auto Imo = std::make_shared<Matrix>("MO ERI Tensor", n1 * n2, n3 * n4);
     double **Imop = Imo->pointer();
 
+#pragma omp parallel for collapse(4) num_threads(nthread_)
```

Comment:
This loop is just doing assignment. I doubt it is CPU bound. Parallellization might actually hurt performance.